recipe/0001-respect-QUARTO_VERSION-set-by-build-scripts.patch

```diff
@@ -0,0 +1,14 @@
+Respect QUARTO_VERSION value set by build scripts
+
+diff --git a/configuration b/configuration
+index 0b6bef3..0239a7d 100644
+--- a/configuration
++++ b/configuration
+@@ -44,7 +44,6 @@ export ALGOLIA_SEARCH_INSIGHTS_JS=2.0.3
+ 
+ 
+ # Quarto Info Version
+-export QUARTO_VERSION=1.3
```

Comment:
Can we drop this patch? The `build.sh` script sets the version after sourcing this patched configuration script. That should override the unpatched value.

https://github.com/conda-forge/quarto-feedstock/blob/main/recipe/build.sh#L25-L28

Reply:
Yes, good call